relay-general/src/protocol/span.rs

```diff
@@ -70,7 +74,12 @@ mod tests {
   "op": "operation",
   "span_id": "fa90fdead5f74052",
   "trace_id": "4c79f60c11214eb38604f4ae0781bfb2",
-  "status": "ok"
+  "status": "ok",
+  "data": {
+    "http": {
+      "query": "is_sample_query=true"
+    }
+  }
```

Comment:
Should we add some random extra fields in `data` and `data.http` here to validate that arbitrary data is still accepted?